chapters/features.txt

```diff
@@ -5451,6 +5451,33 @@ include::{generated}/validity/structs/VkPhysicalDeviceImageProcessingFeaturesQCO
 --
 endif::VK_QCOM_image_processing[]
 
+ifdef::VK_EXT_map_memory_placed[]
+[open,refpage='VkPhysicalDeviceMapMemoryPlacedFeaturesEXT',desc='Structure describing placed memory map features that can be supported by an implementation',type='structs']
+--
+The sname:VkPhysicalDeviceMapMemoryPlacedFeaturesEXT structure is defined
+as:
+
+include::{generated}/api/structs/VkPhysicalDeviceMapMemoryPlacedFeaturesEXT.txt[]
+
+This structure describes the following features:
+
+  * [[features-memoryMapPlaced]] pname:memoryMapPlaced indicates that
+    the implementation supports placing memory maps at client-specified
+    virtual addresses.
+  * [[features-memoryMapRangePlaced]] pname:memoryMapRangePlaced indicates
+    that the implementation supports placing memory maps of a subrange of a
+    memory object at client-specified virtual addresses.
+  * [[features-memoryUnmapReserve]] pname:memoryUnmapReserve
+    indicates that the implementation supports atomically re-reserving
```

Comment:
This text I think should similarly be updated to the "don't unreserve it" language.

Reply:
Also, what's the point in these latter two being features - do these need to be explicitly enabled? Feels like they might make more sense as properties?

Reply:
Do they need to be explicitly enabled?  Probably not.  Are they really properties, though?  I don't think they are.  They both gate functionality.  One says whether or not you can map subranges and the other says whether or not `UNMAP_RESERVE` is supported.  Neither of these feels like a property to me.

Reply:
I mean you're not wrong, but at the same time... eh. I usually plop in a single feature for things like this and a bunch of properties, saves developers having to enable multiple things. It's probably not the biggest deal either way though 🤷‍♀️ - if you want to call this resolved with no change that is a'ok with me.